applications/GeoMechanicsApplication/tests/cpp_tests/test_fluid_pressure.cpp

```diff
@@ -0,0 +1,43 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Gennady Markelov
+//
+
+#include "custom_retention/retention_law.h"
+#include "includes/checks.h"
+#include "testing/testing.h"
+
+using namespace Kratos;
+
+namespace Kratos::Testing
+{
+
+KRATOS_TEST_CASE_IN_SUITE(CalculateFluidPressureGivesCorrectResults, KratosGeoMechanicsFastSuite)
+{
+    Vector N(5);
+    N(0) = 1.0;
+    N(1) = 2.0;
+    N(2) = 3.0;
+    N(3) = 4.0;
+    N(4) = 5.0;
+
+    Vector pressure_vector(5);
+    pressure_vector(0) = 0.5;
+    pressure_vector(1) = 0.7;
+    pressure_vector(2) = 0.8;
+    pressure_vector(3) = 0.9;
+    pressure_vector(4) = 0.4;
```

Comment:
In the same way, you can simplify these assignments:
```suggestion
    pressure_vector <<= 0.5, 0.7, 0.8, 0.9, 0.4;
```

Reply:
Done